### PR TITLE
fix/error handilng ui for blocks

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: packages/plugin-image/web/src/image-component.jsx [hunk 1]
```diff
@@ -168,9 +171,10 @@ class ImageComponent extends React.Component {
           onCaptionChange={this.handleCaptionChange}
           setFocusToBlock={blockProps.setFocusToBlock}
           setComponentUrl={setComponentUrl}
+          t={t}
```
Comment (shaulgo) [inline packages/plugin-image/web/src/image-component.jsx:174]: revert

File: packages/plugin-image/web/src/image-component.jsx [hunk 2]
```diff
@@ -168,9 +171,10 @@ class ImageComponent extends React.Component {
...
         />
         {(this.state.isLoading || componentData?.loading) && this.renderLoader()}
-        {error && <MediaItemErrorMsg error={error} t={t} />}
+        {error && this.renderErrorMessage(error)}
```
Comment (shaulgo) [inline packages/plugin-image/web/src/image-component.jsx:177]: revert

File: packages/plugin-image/web/src/toolbar/inline-buttons.tsx [hunk 3]
```diff
@@ -45,7 +45,7 @@ const createInlineButtons: CreateInlineButtons = ({
     mobile: false,
     tooltipTextKey: 'ImageEditorButton_Tooltip',
     mapComponentDataToButtonProps: componentData => ({
-      disabled: isEmpty(componentData.src),
+      disabled: isEmpty(componentData.src) || !!componentData.error,
```
Comment (shaulgo) [inline packages/plugin-image/web/src/toolbar/inline-buttons.tsx:48]: check what happens during load.

**Review by shaulgo — CHANGES_REQUESTED** [review]

File: packages/plugin-image/web/src/toolbar/inline-buttons.tsx (hunk 3 above)
Comment (talevy17) [reply packages/plugin-image/web/src/toolbar/inline-buttons.tsx:48]: it's disabled during load as is

**Review by shaulgo — APPROVED** [review]

**Pull request merged**